app/js/views/services.jsx

```diff
@@ -1,8 +1,6 @@
 import React from 'components/react';
 
-import NavigationMenu from 'js/views/navigation-menu';
 import ServicesList from 'js/views/services-list';
```

Comment:
nit: So do we want to append `View` to all the views class names? If so, let's rename it to `ServicesListView`.